ui/src/app/lib/atlasmap-data-mapper/components/mapping/mapping.field.detail.component.ts

```diff
@@ -35,45 +35,54 @@ import { MappingModel, FieldMappingPair, MappedField } from '../../models/mappin
         <ng-template #tolTemplate>
             <div class="fieldDetailTooltip">
                 <label class="parentObjectName" *ngIf="displayParentObject()">
-                    <i [attr.class]="getSourceIconCSSClass()"></i>
-                    {{ getParentObjectName() }}
+                    <i [attr.class]="sourceIconCSSClass"></i>
+                    {{ parentObjectName }}
                 </label>
                 <label>{{ getFieldPath() }}</label>
                 <label *ngIf="displayParentObject() && mappedField.field.type">({{ mappedField.field.type }})</label>
                 <div class="clear"></div>
             </div>
         </ng-template>
 
-        <div class='fieldDetail' style="margin-bottom:5px;" *ngIf="mappedField"
-            [tooltip]="tolTemplate" placement="left">
+        <div class='fieldDetail' style="margin-bottom:5px;" *ngIf="mappedField">
```

Comment:
Quotes are not consistent here. The convention for element attributes will be double quotes.